Example/Example/Base.lproj/Main.storyboard

```diff
@@ -131,7 +137,116 @@
+                                                </label>
+                                                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" translatesAutoresizingMaskIntoConstraints="NO" id="rcf-7a-4wd" userLabel="Cover Image View">
+                                                    <rect key="frame" x="89" y="0.0" width="104" height="156"/>
+                                                    <constraints>
+                                                        <constraint firstAttribute="height" constant="156" id="jwl-1h-6K9"/>
+                                                        <constraint firstAttribute="width" constant="104" id="wIn-It-utl"/>
+                                                    </constraints>
+                                                </imageView>
+                                            </subviews>
+                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="calibratedWhite"/>
+                                        </view>
+                                        <constraints>
+                                            <constraint firstItem="9Sf-Jp-het" firstAttribute="top" secondItem="rcf-7a-4wd" secondAttribute="bottom" constant="5" id="8h3-F2-pJf"/>
+                                            <constraint firstItem="rcf-7a-4wd" firstAttribute="centerX" secondItem="T99-sD-NG3" secondAttribute="centerX" id="ZoZ-3g-8fT"/>
+                                            <constraint firstItem="rcf-7a-4wd" firstAttribute="top" secondItem="T99-sD-NG3" secondAttribute="topMargin" constant="-8" id="jgc-Nu-XqZ"/>
+                                            <constraint firstItem="9Sf-Jp-het" firstAttribute="centerX" secondItem="rcf-7a-4wd" secondAttribute="centerX" id="quT-yW-qfh"/>
+                                            <constraint firstItem="9Sf-Jp-het" firstAttribute="width" secondItem="rcf-7a-4wd" secondAttribute="width" id="yPQ-cT-Hdh"/>
+                                        </constraints>
+                                        <connections>
+                                            <outlet property="comicNameLabel" destination="9Sf-Jp-het" id="FPH-ox-Cce"/>
+                                            <outlet property="coverImageView" destination="rcf-7a-4wd" id="i3G-43-XUE"/>
+                                        </connections>
+                                    </collectionViewCell>
+                                </cells>
+                                <collectionReusableView key="sectionHeaderView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" reuseIdentifier="SeriesDetailCollectionHeaderViewReusableIdentifier" id="0Tn-pl-Jzw" customClass="SeriesDetailCollectionHeaderView" customModule="Example" customModuleProvider="target">
```

Comment:
It's ok to do this with an header, is more the Android way, generally we define this page as a scroll view, with Image + label + CollectionView(configured with no scroll)